L1Trigger/Phase2DAQProducer/interface/DTCAssembly.h

```diff
@@ -0,0 +1,44 @@
+#ifndef DTCASSEMBLY_H
+#define DTCASSEMBLY_H
+
+#include <vector>
+#include "FWCore/Utilities/interface/Exception.h"
+#include "./DTCUnit.h"
+
```

Comment:
All classes must have a comment at the start of the header explaining what they represent.

Reply:
done.